File: 2022-2023/sourcecode/lab/lab11/VeryNobleAppJava/app/src/main/java/pl/udu/uwr/pum/verynobleappjava/data/laureateresponse/laureate/Place.java
package pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.laureate;

import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.affiliation.City;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.affiliation.CityNow;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.affiliation.Continent;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.Country;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.CountryNow;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.LocationString;

public class Place {
    public City city;
    public Country country;
    public CityNow cityNow;
    public CountryNow countryNow;
    public Continent continent;
    public LocationString locationString;
}

File: 2022-2023/sourcecode/lab/lab11/VeryNobleAppKotlin/app/src/main/java/pl/udu/uwr/pum/verynobleappkotlin/data/laureateresponse/laureate/Place.kt
package pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.laureate

import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.Country
import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.CountryNow
import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.LocationString
import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.affiliation.City
import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.affiliation.CityNow
import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.affiliation.Continent

data class Place(
    val city: City,
    val cityNow: CityNow,
    val continent: Continent,
    val country: Country,
    val countryNow: CountryNow,
    val locationString: LocationString
)